mantidproject/mantid
# 33704 add new documentation on crystal fields

### Description of work
- Included missing documentation on Crystal Fields pages
-  Theory and equations provided by @mducle 

#### Summary of work
- Included new information on Crystal Field Python Interface page.
- Included new information on CrystalFieldSpectrum page.
- Included new information on concepts Crystal Field Theory page.

#### Purpose of work
- To add missing information, requested in issue #33704 by @mducle.

#### Further detail of work
- New information added is the definition of external and molecular magnetic field contributions for the CrystalField interface.
- Added very relevant note that external fields must always be fixed by ties before performing a fit.
- By default, external and molecular fields are free to vary, and need to be tied manually. Maybe change this in the future?

### To test:
- Read through new added documentation and check that it makes sense scientifically (for Duc)
- Build docs and check that formatting looks good and links work.

*I believe this does not require release notes?*


Fixes #33704 
---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
